commit aebc3ca19063d68b76bcaaca81558d4f180c61b0
Author: Jue Wang <juew@google.com>
Date:   Fri Jun 10 10:11:33 2022 -0700

    KVM: x86: Enable CMCI capability by default and handle injected UCNA errors
    
    This patch enables MCG_CMCI_P by default in kvm_mce_cap_supported. It
    reuses ioctl KVM_X86_SET_MCE to implement injection of UnCorrectable
    No Action required (UCNA) errors, signaled via Corrected Machine
    Check Interrupt (CMCI).
    
    Neither of the CMCI and UCNA emulations depends on hardware.
    
    Signed-off-by: Jue Wang <juew@google.com>
    Signed-off-by: Paolo Bonzini <pbonzini@redhat.com>
    Message-Id: <20220610171134.772566-8-juew@google.com>
    Signed-off-by: Paolo Bonzini <pbonzini@redhat.com>

diff --git a/arch/x86/kvm/vmx/vmx.c b/arch/x86/kvm/vmx/vmx.c
index 4002f6cc179d..c30115b9cb33 100644
--- a/arch/x86/kvm/vmx/vmx.c
+++ b/arch/x86/kvm/vmx/vmx.c
@@ -8299,6 +8299,7 @@ static __init int hardware_setup(void)
 	}
 
 	kvm_caps.supported_mce_cap |= MCG_LMCE_P;
+	kvm_caps.supported_mce_cap |= MCG_CMCI_P;
 
 	if (pt_mode != PT_MODE_SYSTEM && pt_mode != PT_MODE_HOST_GUEST)
 		return -EINVAL;